scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1182,6 +1208,7 @@ def get_cluster_name( self ):
     def query_api( self, path, pretty=0 ):
         """ Queries the k8s API at 'path', and converts OK responses to JSON objects
         """
+        raise Exception( "K8s api queries disabled" ) # echee
```

Comment:
Are you guys OK that we leave this in here to be 100% certain we're not making any API requests?

Reply:
I say yes, for these reasons.

1. We'll get a searchable message in agent.log that we can track and fix
2. The big problem for Bukalapak is the CPU and network on master so I think we should try and avoid hitting the API at all for this.  If this causes us to miss out on a few logs, I feel that's a better option than having a high CPU and network load caused by unexpected queries.

Presumably we haven't seen this message at all when running the .6 on the test cluster?  If so I think we're ok.

Reply:
I think if we run this build on the load test cluser and this doesn't trip, I'm 100% certain.  I'm about 99% certain now.  But, maybe we should just replay this with a log message for now.  I'm open to that.  If there was some trouble we hit in tomorrow's run, we'd have to do a code change to remove this.

Reply:
OK I will remove the exception raise and replace with a logging statement.   Since we have run this code on a 200 node vs 1 master cluster and it's clearly not pounding the master, it should be safe.